frontend/packages/ceph-storage-plugin/src/components/ocs-install/install-page.tsx

```diff
@@ -121,7 +121,7 @@ const InstallCluster: React.FC<InstallClusterProps> = ({ match }) => {
           </div>
         )}
         {(isIndependent === false || mode === MODES.CONVERGED) && (
```

Comment:
```suggestion
        {(isIndependent === false || mode === MODES.CONVERGED) && csv && (
```
Or else the old bug will start coming

Reply:
Ack